### PR TITLE
RecommenderData Table update

### PR DESCRIPTION
**Pull request by jasonjabbour**

### CODE REVIEW

File: src/recommender.py [hunk 1]
```diff
@@ -106,76 +162,115 @@ def _send_action(self, speaker_id, action):
...
+        dbr.close()
+
+
+      except:
         err = "Webbrowser Error"
 
 
-    db.close()
-    return empathid, err
+    return err, empathid, action
```
Comment (aobo-y) [inline src/recommender.py]: why return the `action`, i thought `action` is what we pass in as the arguments

File: src/recommender.py [hunk 2]
```diff
@@ -106,76 +162,115 @@ def _send_action(self, speaker_id, action):
...
       androidid = 'db7d3cdb88e1a62a'
       alarm = 'true'
 
       # sending action to phone
       try:
-        #send prequestion 22
-        url = phone_url + '/?q={%22id%22:%22' + str(speaker_id) + '%22,%22c%22:%22startsurvey%22,%22suid%22:%22' + '22' + '%22,%22server%22:%22' + server_url + '%22,%22androidid%22:%22' + androidid + '%22,%22empathid%22:%22' + pre_empathid + '%22,%22alarm%22:%22' + alarm + '%22}'
-        #webbrowser.open(url)  # to open on browser
+        # send prequestion 22
+        url = phone_url + '/?q={%22id%22:%22' + str(speaker_id) + '%22,%22c%22:%22startsurvey%22,%22suid%22:%22' + '22' + '%22,%22server%22:%22' + \
```
Comment (aobo-y) [inline src/recommender.py]: just a tiny suggestion.... do you find these encoded strings are actually so hard to read, like these `%22`
you want to refer [this api](https://stackoverflow.com/questions/8905864/url-encoding-in-python) to encode a readable one

**Comment by aobo-y** [comment]
@jasonjabbour  your pull request has conflicts that i can not merge
I checked your commit history, seems you forget to pull from my repo as we discussed (the pull & push cycle to synchronise our work).

you may want to 
- `git remote add upstream git@github.com:aobo-y/caregiver_recomm.git` to set my repo as your upstream
- `git pull upstream master` to get my latest code
- you may see some conflicts, see if you can resolve them
- then `git commit` to commit your merge
- `git push` to push the codes without conflicts to your repo so that I can merge this

**Comment by jasonjabbour** [comment]
recommender.py has been updated to fulfil the requirements we spoke about on Friday!.

**Pull request merged**